configs/webpack-config-compass/src/loaders.ts

```diff
@@ -82,7 +82,7 @@ export const cssLoader = (_args: ConfigArgs) => ({
           plugins: [
             [
               require.resolve('postcss-preset-env'),
-              { browsers: `electron ${electronVersion}` },
+              { browsers: 'electron 6.1' },
```

Comment:
`postcss-preset-env` browser list wasn't finding electron 13.5
I don't think this is a blocker since building for electron 6 browser env is ok, but we should update it.

Similairly in many of the packages there are `.browserlistrc` files which define the browser target for babel. Currently they are all `Electron >= 6` we should update them to `Electron >= 13`, but it will take some work updating babel deps and config I think. Could happen post merge of this even, but we should do it sooner vs later.